### PR TITLE
Uat elevation mixins update gh

### PR DESCRIPTION
**Pull request by RachelRVasquez**

### CODE REVIEW

File: src/html/elevations/elevations.html [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<div class="ds-elevation-remove"></div>
```
Comment (pmmck) [inline src/html/elevations/elevations.html]: @RachelRVasquez why was this flat example removed?

Comment (RachelRVasquez) [inline src/html/elevations/elevations.html]: @pmmck We currently don't have a class called ds-elevation-remove in the DS. This is being used only as a mixin currently. Do we want to create that class?

Comment (pmmck) [inline src/html/elevations/elevations.html]: @RachelRVasquez it seems like a nice thing to have and we could include it in the example to show the full range of flat to elevation-5.  Thoughts?

Comment (RachelRVasquez) [inline src/html/elevations/elevations.html]: @pmmck @fuhton This PR has been updated.

**Comment by pmmck** [comment]
@RachelRVasquez @fuhton does this PR fix the broken layout on the Layout.html page?

https://www.evernote.com/l/ABfdJI9U0fJLHIJpxzOW048gN-wgTnT4XUo

Or should I open a new issue?

**Comment by fuhton** [comment]
@pmmck This resolved this issue.

**Pull request merged**